src/main/java/nextstep/fp/Lambda.java

```diff
@@ -27,21 +28,28 @@ public void run() {
     }
 
     public static int sumAll(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            total += number;
-        }
-        return total;
+        return numbers.stream()
```

Comment:
`sumAll(List<Integer> numbers, Conditional c)` 메서드를 사용해, `sumAllXXX` 메서드에서 발생하는 stream 관련한 중복 로직을 줄일 수 있겠네요! 

```suggestion
        return sumAll(numbers, number -> true);
```